### ISSUE TITLE
PIL

### ISSUE THREAD

**Issue by m00n-err0r** [brython-dev/brython#2107]
Hello! Is there a way to use PIL (Python Imaging Library)?

**Comment by PierreQuentel** [comment]
I don't think so, the source code of Pillow (a fork of PIL) is full of files written in C, and Brython only works with pure Python scripts.
